lib/paper_trail/version_concern.rb

```diff
@@ -1,4 +1,5 @@
 require 'active_support/concern'
+require 'json'
```

Comment:
This is unnecessary because the `to_json` method is already provided by [this requiring of `active_support/json`](https://github.com/airblade/paper_trail/blob/master/lib/paper_trail/serializers/json.rb#L1)